cmd/create/cluster/cmd.go

```diff
@@ -2092,7 +2092,10 @@ func run(cmd *cobra.Command, _ []string) {
 		isHostedCP,
 		multiAZ)
 
-	if autoscaling {
+	var clusterAutoscaler *clusterautoscaler.AutoscalerArgs
+	if !autoscaling {
```

Comment:
Does this mean the global autoscaler will be enabled only when the machine pool/node pool autoscaling is enabled?

(I'm thinking in side effects for the hosted cluster area)

Reply:
HCP is out of scope at the moment for the cluster-autoscaler

Reply:
cool, didn't know you're working on that

refer to the DDR at https://issues.redhat.com/browse/OCM-70, it has most of the relevant information for the design of the feature
basically speaking, on the backend we had to make those two constructs (machine-pools, cluster-autoscaler) independent from each other

Reply:
It's not a big change, it can reuse the whole design and apply it to the provisioner for hosted clusters.

It's experimental, but there is a draft with day 1 enabled under this:
https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/merge_requests/6442

The only relevant change is that not all fields apply to the hosted clusters and there should be a validation for those cases.

But it should be transparent for this current OCM-70 work.